src/python-bindings/htcondor2/submit.cpp

```diff
@@ -474,6 +474,11 @@ set_dag_options( PyObject * options, DagmanOptions& dag_opts) {
             return false;
         }
 
+        std::string err;
+        if( !dag_opts.checkMutualExclusion(k, err) ) {
+            PyErr_SetString(PyExc_RuntimeError, err.c_str());
+            return false;
+        }
```

Comment:
@Todd-L-Miller I wanted to get your opinion here before merging... should this be a RuntimeError?

Reply:
I'm not thrilled by how generic `RuntimeError` is, but I agree that there isn't a better built-in and that `HTCondorException` seems misleading as well.  Let's leave as a `RuntimeError` for now but not document it as part of the API yet, so that there's less of a barrier to changing it later if we have a better idea.